docs/downloads/install.sh

```diff
@@ -115,7 +115,7 @@ URL="https://github.com/fermyon/spin/releases/download/${VERSION}/${FILE}"
 
 # Download file, exit if not found - e.g. version does not exist
 fancy_print 0 "Step 1: Downloading: ${URL}"
-wget -q $URL || (fancy_print 1 "The requested file does not exist: ${FILE}"; exit 1)
+curl -sOL $URL || (fancy_print 1 "The requested file does not exist: ${FILE}"; exit 1)
```

Comment:
This doesn't capture the "file does not exists" scenario. I'm getting an invalid archive downloaded.

E.g., `./install.sh -v rubbish`

I think you would have to add `--fail` to curl to have it return a non-zero exit code.

Reply:
You're right :) added the `fail` flag